### PR TITLE
Add DataRow

### PR DESCRIPTION
**Pull request by patrickkim**

### CODE REVIEW

File: src/molecules/DataRow/data_row.module.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+@import 'assets/stylesheets/global';
+
+
+.data-row {
+  // Place your component specific scss here.
```
Comment (jmcolella) [inline src/molecules/DataRow/data_row.module.scss]: Can you remove this comment?

File: src/molecules/DataRow/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    PropTypes.string,
+    PropTypes.number,
+  ]).isRequired,
+  /**
+   * optional helper text describing this data.
+   */
+  description: PropTypes.string,
+};
+
+DataRow.defaultProps = {
```
Comment (jmcolella) [inline src/molecules/DataRow/index.js]: Do we need this if there aren't any default Props?

File: src/molecules/DataRow/index.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+import styles from './data_row.module.scss';
+
+function DataRow(props) {
+  const {
+    className,
+    variant,
+    label,
+    value,
+    description,
```
Comment (jmcolella) [inline src/molecules/DataRow/index.js]: stray comma here

File: src/molecules/DataRow/index.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  } = props;
+
+  return (
+    <div className={classnames(styles['data-row'], styles[variant], className)}>
+      <span className={classnames(styles['label'])}>
+        { label }
+        { description && <Tooltip className={styles['tip']}>{ description }</Tooltip> }
```
Comment (jmcolella) [inline src/molecules/DataRow/index.js]: Looking at the screenshot on Clubhouse (https://app.clubhouse.io/policygenius/story/1861/datarow), it seems like a `label` can have a `tooltip` and it doesn't seem like this implementation will allow for that. Am I thinking about this wrong?

Comment (patrickkim) [inline src/molecules/DataRow/index.js]: It would if it's passed in as a description prop

**Pull request merged**